docker-compose.dev.yml

```diff
@@ -23,10 +23,11 @@ services:
     volumes: *sbt-volumes
     stdin_open: true # docker run -i
     tty: true # docker run -t
+    restart: always
     ports:
       - 9000:9000
       - 8457:8457
-    command: -jvm-debug "*:8457" ~run -Dconfig.file=conf/application.dev.conf
+    command: -jvm-debug "*:8457" ~run -Dsbt.offline -Dconfig.file=conf/application.dev.conf
```

Comment:
Can you add comment clarifying why sbt.offline is helpful in this scenario? Will it prevent pulling dependencies from maven repo?

Reply:
Yes - I'm not 100% sure it will always skip them, but it anecdotally seems to work

Reply:
How will it work in the case when someone adds are new dependency locally and runs `run-dev` expecting that it will fetch it and compile app with it? Will the engineer have to remove `sbt.offline`?

Reply:
It looks like it's just a hint, and will download the dependancy anyway